config/default.yml

```diff
@@ -4,3 +4,6 @@ collectors:
     ansible-tower: 2
     azure: 2
     openshift: 2
+sync:
+  poll_time_seconds: 10 # seconds
+  scheduled_event_hours: 1 # hours
```

Comment:
Do you think we will want this to be more frequent in the future?  We usually specify numbers in the base unit (time in seconds)

Reply:
I think we will want this to be much less frequent, like once per day/week

Reply:
@bdunne it would be good to know how many sources are expected in production. If something like milions, then full sync should be disabled or only on demand.

Reply:
I don't think we have any estimates yet.  We''ll have to adjust as load increases.